dmutils/forms/dm_fields.py

```diff
@@ -0,0 +1,65 @@
+import wtforms.fields
+
+import dmutils.forms.fields
+
+from .mixins import DMFieldMixin
+
+
+__all__ = ['DMBooleanField', 'DMDecimalField', 'DMEmailField', 'DMHiddenField',
+           'DMIntegerField', 'DMRadioField', 'DMStringField', 'DMStripWhitespaceStringField']
+
+
+class DMBooleanField(DMFieldMixin, wtforms.fields.BooleanField):
+    pass
```

Comment:
should we include all of these classes that don't do anything?

Reply:
So I included specialisations for all classes that I saw being used in the in the app codebase. Although the class body is empty, the fact that it inherits from the mixin means that the new class gets all the features of  https://github.com/alphagov/digitalmarketplace-utils/blob/ldeb-wtforms-dmfields/dmutils/forms/dm_fields.py.

I'll add a docstring to the module that explains this.